### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by MarkG33** [padams/Open-Web-Analytics#323]
So I installed OWA and it's not working.

I can see the script in my pages source but for some reason it's not working.

Anyone have any ideas as to why it doesn;t work?

Thanks

**Comment by themancalledjakob** [comment]
You should add more information to your issue than this.
What exactly is not working?
At which step of the installation process were you?
For example, did you run the install.php and get to the administration panel?
Are you trying to implement the javascript or php tracker?
Is there any error message in the developer console?

**Comment by destinationav** [comment]
It's not working for me either.  I followed all instructions step by step but it doesn't log IP, Browser, Visits, Clicks, Time on Page
![capture](https://user-images.githubusercontent.com/43298995/45588251-0071d000-b8cf-11e8-9c7c-bd86bfb58cee.PNG)
![capture2](https://user-images.githubusercontent.com/43298995/45588252-0071d000-b8cf-11e8-9a39-b187ea473254.PNG)

 etc.

I have reinstalled it several times to no avail.  When I look in AWStats it is registering as normal and I can see all of my visitors and their IP addresses.

What can be the problem?  Should I install a previous version?
